### ISSUE TITLE
RE: error about the Round Robin Database (RRD) file

### ISSUE THREAD

**Issue by bdmu** [jexp/neo4j-clean-remote-db-addon#4]
Hello,

This issue "https://github.com/jexp/neo4j-clean-remote-db-addon/issues/2" doesn't seem to be totally fixed:
there is no more error in log, but the graph showing the number of node and properties is not updating in the webadmin interface:
the count go down to 0 and doesn't increase anymore when creating new nodes.

It would be very nice if you could fix this error.

Thanks in advance

Best regards.
Christophe

**Comment by ghost** [comment]
It seems to be a bug that the handling of rrd is invalid when starting the database.
Debgging  ServerStatistics.coffee, It checked that all montor datas were invalid.
For example, if the number of node is 5, "node_count" is -5.

**Comment by jexp** [comment]
Yep, that is an unfortunate coincidence as deleting the db behind the back of RRD let's the RRD become invalid. Haven't found a workaround around this yet (except a full server restart but then the value of this plugin is in vain).

**Comment by ghost** [comment]
The MonitorService constructor has an argument of rrd.
Isn't the old rrd instance injected even if new rrd instance is set to Database instance
before starting up the database?

Actually I tried to do so, but the handling of rrd was invalid.
